install

```diff
@@ -31,22 +59,29 @@ pip install -e buzzmobile
 pip install opencv-python==3.1.0.3
 
 # Unfortunately, tensorflow isn't availible on pypi, so we have to pull it from this very readable link
-pip install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0rc1-cp34-cp34m-linux_x86_64.whl
+#pip install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0rc1-cp34-cp34m-linux_x86_64.whl
+pip install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.12.1-cp27-none-linux_x86_64.whl
```

Comment:
see issue #154. I'd wait on it a bit, because Tensorflow 1.0 is still in release candidate stage.